dm/syncer/error.go

```diff
@@ -61,6 +65,139 @@ func isDropColumnWithIndexError(err error) bool {
 			strings.Contains(mysqlErr.Message, "with tidb_enable_change_multi_schema is disable"))
 }
 
+// getDDLStatusFromTiDB retrieves the synchronizing status of DDL from TiDB
+// hence here db should be TiDB database.
+func getDDLStatusFromTiDB(ctx context.Context, db *sql.DB, ddl string, createTime int64) (string, error) {
+	rowNum := 10
+	count := 0
+	rowOffset := 0
+
+	showJobsLimit := fmt.Sprintf("ADMIN SHOW DDL JOB QUERIES LIMIT 10 OFFSET %d", rowOffset)
```

Comment:
with line 144: can we query `ADMIN SHOW DDL JOB QUERIES` in one place?

Reply:
I don't think it's very convenient. The ```ADMIN SHOW DDL JOB QUERIES``` in line 75 can be considered as the base for these queries and should be executed first, and the one in line 144 serves the purpose of expanding the slice. The latter one is inside 2 for loops, it may not be easy to combine those two queries.